_data/authors.yml

```diff
@@ -0,0 +1,26 @@
+rohitrox:
+  username: rohitrox
+  name: Rohit Joshi
+  location: Kathmandu, Nepal
+  twitter-username: "rohitroxxxxx"
+  github-username: "RohitRox"
+  bio: "Proud to be a kapil fan nd a gautam rode fan met gautam rode on 5th july'2014 and he followed me on 7th
```

Comment:
@RohitRox

Reply:
@RohitRox  bro, Bio and photo link is copied from twitter, Make sure that is yours :)

Reply:
nop definitely not his 🤣 
see above comment for Rohit's twitter url